yarn.lock

```diff
@@ -3064,6 +3064,7 @@ __metadata:
   version: 0.0.0-use.local
   resolution: "@fern-api/openapi-generator@workspace:generators/openapi"
   dependencies:
+    "@fern-api/generator-commons": "workspace:*"
     "@fern-fern/generator-exec-sdk": 0.0.675
```

Comment:
i think a successful PR here means that `"@fern-fern/generator-exec-sdk"` is not used in any of the generators.

Reply:
what would it take to get to that?